source/extensions/filters/http/ext_proc/client_impl.cc

```diff
@@ -12,6 +12,10 @@ ExternalProcessorClientImpl::ExternalProcessorClientImpl(
     const envoy::config::core::v3::GrpcService& grpc_service, Stats::Scope& scope)
     : client_manager_(client_manager), grpc_service_(grpc_service), scope_(scope) {}
 
+ExternalProcessorClientImpl::~ExternalProcessorClientImpl() {
+  ENVOY_LOG(trace, "~ExternalProcessorClientImpl");
```

Comment:
Revert those?

Reply:
I took out a lot of new trace statements, but I left these in because, previously in debugging gRPC client stuff, I needed them to track how destructors were being invoked, and I felt like since I put them in and took them out twice already, I should leave them this time. I'm happy to remove them, though, if having destructors that do nothing but log is just too much code bloat.

Reply:
These should at least be more informative but not just destructor name here. something like `ext_proc: destroying client`, and attach a stream id.

Reply:
Thanks -- I looked in to including the stream info and it's not worth the trouble for the theoretical case where we'll need debugging in the future, so I rolled back all the files where I just added tracing. Now the only changes are the ones necessary to fix the flaky test.